### ISSUE TITLE
[FPGA] MASK_ROM/ROM_EXT exec_environments require info flash page initialization

### ISSUE THREAD

**Issue by moidx** [lowRISC/opentitan#23038]
## Issue 

Currently, the flash_ctrl and keymgr may raise alerts due to the initial info flash configuration when booting with the `mask_rom` and `rom_ext` targets.

* The keymgr requires the Owner and Secret owner pages to be initialized.
* Ownership transfer requires the owner block to be initialized.

## Options

1. Modify all environments using the real `rom_ext` so that there is an initial bootstrap operation with the personalization image to initialize all required info partitions. Applicable to all FPGA configurations.
2. Register a weak functions in the `rom_ext` to initialize the owner partition for silicon targets already deployed in various environments. 

CC: @cfrantz @timothytrippel  @jwnrt

**Comment by timothytrippel** [comment]
I think it may be more efficient to go with option 2, since it could be much lighter weight than running a full personalization on every boot, since we really just need to program those two pages.

**Comment by jwnrt** [comment]
Related https://github.com/lowRISC/opentitan/issues/21225

**Comment by timothytrippel** [comment]
yes, another option is splicing, the only downside with splicing is it will require a lot more infrastructure than a weak function in the ROM_EXT
